### ISSUE TITLE
Failsafe setup

### ISSUE THREAD

**Issue by Eurylokhos** [FrSky-OS/discussion#48]
Failsafe setup is nice the way it is, however, a button that would set the failsafe on channels to what the sticks and switches are currently set at greatly simplifies things and would make a fool proof setup for less advanced users.

**Comment by MIKE-DAILEY** [comment]
As an option though, right.  I like the way it works now that each channel can be independently set with "preset", "hold" or "none".

**Comment by Eurylokhos** [comment]
Yes, I said that it is good how it is done now. It would be a nice addition to add a button to set fail safe. Jeti handles this especially well in the V3 and newer firmware. It has a screen similar to how the Horus is setup and a button to set fail safe positions. Hitting that button sets the current stick and switch values to any channels that have been selected to go to a specific position, and leaves the rest alone.

**Comment by yellowbird911** [comment]
The servo position window should not be editable when anything other than Custom is selected.  I agree that most radios offer a way to set fail safe by simply setting the sticks and switches to the position you want and pushing a button or menu button.  I like the OpenTx type but most of us are experienced users, the new user may not find this menu as friendly.
